core/server/helpers/asset.js

```diff
@@ -2,15 +2,17 @@
 // Usage: `{{asset "css/screen.css"}}`, `{{asset "css/screen.css" ghost="true"}}`
 //
 // Returns the path to the specified asset. The ghost flag outputs the asset path for the Ghost admin
-var proxy = require('./proxy'),
+const proxy = require('./proxy'),
     _ = require('lodash'),
```

Comment:
Do we want to update this for the specific lodash module (i.e. `lodash/get`)?

Reply:
That can be done

Reply:
Yeah why not - but then it must be a rule for every PR part of https://github.com/TryGhost/Ghost/issues/9589 :)

Reply:
Makes sense! I added a checklist to the bottom of OP which can be used for these PRs